src/ui/components/inputs/InputContainer.tsx

```diff
@@ -56,29 +53,26 @@ const StyledView = styled(View)<{
     if (isDisabled) {
       return undefined
     }
-    return isError ? theme.colors.error : theme.colors.greyDark
+    if (isFocus) {
+      return theme.colors.black
+    }
+    return isError ? theme.colors.error : theme.colors.greySemiDark
   }
 
-  const focusRules =
-    Platform.OS === 'web' && focusOutlineColor
-      ? customFocusOutline(theme, focusOutlineColor, isFocus)
-      : { borderColor: theme.colors.primary }
-
   return {
     height: getHeightValue(),
     width: '100%',
     flexDirection: 'row',
     borderStyle: 'solid',
-    borderWidth: '1px',
+    borderWidth: isFocus ? getSpacing(0.5) : getSpacing(0.25),
```

Comment:
Pourquoi ne pas utiliser `outline` pour avoir l'effet que tu veux ? La ça fait changer le `border` et le `padding` alors que si tu ajoutes un outline, ça ne bouge pas le composant.

Reply:
J'ai l'impression qu'`outline` ne marche pas en native, ça marche chez toi ?

Reply:
Tu as essayé ça ? https://stackoverflow.com/questions/62390508/change-border-color-of-textinput-when-focused-in-react-native-web-expo/62431191#62431191

Il ne faut pas `outline` tout seul mais bien détailler (outline-color, outline-width, etc.) comme dans 'useFocusOutline` ?